app/Presenters/Admin/LandingPagePresenter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Presenters\Admin;
+
+use App\Presenters\BasePresenter;
+
+class LandingPagePresenter extends BasePresenter
+{
+    public function introBlocks()
+    {
+        return [
+            [
+                'type' => 'text',
+                'content' => '<p>' . $this->entity->exhibition_history_intro_copy . '</p>'
```

Comment:
You don't need to use `$this->visit_intro` here? I wonder if:

* This should check the type of landing page and use the field necessary for that specific page type
* Or if we should create a single optional `intro` field that can be used by any landing page type

Reply:
I like the idea of a simple intro field for each type. I'll add that under the content section below the type selector